### PR TITLE
Test Improvements and Refactoring

### PR DESCRIPTION
**Pull request by powerhousefrank**
Task List:

- [x] Python Test Removal
- [x] Test Improvements
- [ ] Test Refactoring
- [ ] Test Formatting

### CODE REVIEW

File: tests-js/EscrowReward/state.test.js [hunk 1]
```diff
@@ -130,8 +127,8 @@ async function checkStatus(
  * @return [SOV Balance, Reward Token Balance].
  */
 async function getTokenBalances(addr, sovContract, lockedSOVContract) {
-	let sovBal = await sovContract.balanceOf(addr);
-	let rewardBal = await lockedSOVContract.getLockedBalance(addr);
+	let sovBal = (await sovContract.balanceOf(addr)).toNumber();
+	let rewardBal = (await lockedSOVContract.getLockedBalance(addr)).toNumber();
```
Comment (santosinarp) [inline tests-js/EscrowReward/state.test.js:131]: Sometimes, using toNumber() is not really accurate if the number that is being converted is too big, e.g 100e18 (depends on the processor), so I suggest converting it to string (toString()) which I believe would be more accurate.

File: tests-js/EscrowReward/anyone.test.js [hunk 2]
```diff
@@ -119,6 +115,37 @@ contract("Escrow Rewards (Any User Functions)", (accounts) => {
 		await expectRevert(escrowReward.depositTokensByMultisig(zero, { from: userOne }), "Only Multisig can call this.");
 	});
 
+	it("No one should be able to withdraw unless the Release Time has not set (i.e. Zero).", async () => {
+		let oldSOVBal = await sov.balanceOf(multisig);
+		await escrowReward.withdrawTokensByMultisig(constants.ZERO_ADDRESS, { from: multisig });
+		let newSOVBal = await sov.balanceOf(multisig);
+		let value = newSOVBal - oldSOVBal;
```
[the remark this answers is not in the document]
Comment (powerhousefrank) [reply tests-js/EscrowReward/anyone.test.js]: Solved in commit 1296cfc

File: tests-js/EscrowReward/creator.test.js [hunk 3]
```diff
@@ -132,47 +128,27 @@ contract("Escrow Rewards (Creator Functions)", (accounts) => {
...
-		await sov.mint(creator, value);
-		await sov.approve(escrowReward.address, value, { from: creator });
-		await escrowReward.depositTokens(value, { from: creator });
-		await escrowReward.changeStateToHolding({ from: multisig });
+		await escrowReward.updateReleaseTimestamp(currentTimestamp() + 100, { from: multisig });
+
+		let oldSOVBal = await sov.balanceOf(multisig);
 		await escrowReward.withdrawTokensByMultisig(constants.ZERO_ADDRESS, { from: multisig });
+		let newSOVBal = await sov.balanceOf(multisig);
+		let value = newSOVBal - oldSOVBal;
```
[the remark this answers is not in the document]
Comment (powerhousefrank) [reply tests-js/EscrowReward/creator.test.js]: Solved in commit 1296cfc

File: tests-js/EscrowReward/state.test.js (hunk 1 above)
Comment (powerhousefrank) [reply tests-js/EscrowReward/state.test.js]: Solved in commit 1296cfc

**Review by santosinarp — APPROVED** [review]

**Pull request merged**